### PR TITLE
Load Consistency

### PR DESCRIPTION
**Pull request by Huddie**

### CODE REVIEW

**Comment by Huddie** [comment]
Addresses #2097

File: Classes/Section Controllers/LoadMore/LoadMoreCell.swift [hunk 1]
```diff
@@ -42,14 +49,14 @@ final class LoadMoreCell: UICollectionViewCell {
...
-        didSet {
-            if isSelected {
-                label.alpha = 0.5
-            } else {
-                label.alpha = 1
-            }
+    
+    func configure(loading: Bool) {
+        label.isHidden = loading
+        if loading {
```
Comment (jdisho) [inline Classes/Section Controllers/LoadMore/LoadMoreCell.swift]: you can make this shorter

```swift 
loading ? activity.startAnimating() : activity.stopAnimating()
```

Comment (Huddie) [reply Classes/Section Controllers/LoadMore/LoadMoreCell.swift]: I actually just merged this from the IssueNeckLoadMore. Personality I don’t feel ternary vs full if-else makes a difference but I can switch it

Comment (jdisho) [reply Classes/Section Controllers/LoadMore/LoadMoreCell.swift]: Functionaly is the same, but is shorter.

**Comment by Huddie** [comment]
If someone who has a better overall knowledge of the codebase knows other places where there are load more let me know. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Should be used in the base list VC, base list 2 VC, and issues for sure (sorry for short names, on mobile). I _think_ that would cover everything.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Huddie** [comment]
Okay thanks! I’ll go ahead and see if those are using their own or using LoadMoreCell 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Huddie** [comment]
@rnystrom I think I nabbed them all now. There was one in BaseViewController, nice catch!

**Pull request merged**